src/cascadia/TerminalConnection/AzureConnection.cpp

```diff
@@ -25,6 +25,15 @@ using namespace winrt::Windows::Security::Credentials;
 
 namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
 {
+    // This file only builds for non-ARM64 so we don't need to check that here
+    // This function exists because the clientID only gets added by the release pipelines
+    // and is not available on local builds, so we want to be able to make sure we don't
+    // try to make an Azure connection if its a local build
+    bool AzureConnection::IsAzureConnectionAvailable()
+    {
+        return (AzureClientID != L"0");

```

Comment:
Actually, out of curiosity, how is `AzureClientID` set? Is there like a global `extern` variable we're setting somewhere in a file that's only included in the release pipeline build? An environment variable?  Some other dark magic we're not allowed to discuss externally?

Reply:
@DHowett could answer this better

Reply:
Stomped in by the build :)